content/blog/2016/2016-12-20-jenkins-puppet-enterprise-plugin.adoc

```diff
@@ -258,7 +258,7 @@ since data promotion almost always involves changing Hiera files and committing
 to version control.
 
 The plugin exposes an HTTP API endpoint that Hiera can query using the
-hiera-http backend. With the backend configured on the Puppet master(s),
+hiera-http backend. With the backend configured on the Puppet controller(s),
```

Comment:
Did anyone read any of this before it was merged?

I'm done.

Reply:
I will fix that

Reply:
@markyjackson-taulia Any news here?

Reply:
Will have this in today. I was hospitalized for a spell